### PR TITLE
Updated scripting, CRDs, and README

### PR DESCRIPTION
**Pull request by nathan-weinberg**
Changes made:
- Decoupled Loki and Network Observability Operator installations for increased user flexibility
- Updated README
- Changed flowcollector CRDs `authToken` for loki from `HOST` to `FORWARD`
- Renamed `clusterRoleBinding` files to reflect new authToken 
- Added `netobserv_touchstone_rules.yaml`
- Unified flowcollector CRDs

### CODE REVIEW

**Review by memodi — APPROVED** [review]
just one minor question

File: scripts/queries/netobserv_touchstone_rules.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+- json_path: ["metric_name.keyword", "ebpfMemoryUsageWorkingSet", "metric_name.keyword", "ebpfMemoryUsageWorkingSet", "avg(value)"]
```
[the remark this answers is not in the document]
Comment (nathan-weinberg) [reply scripts/queries/netobserv_touchstone_rules.yaml:1]: this is WIP, need to discuss this in brainstorming today

[the remark this answers is not in the document]
Comment (nathan-weinberg) [reply scripts/queries/netobserv_touchstone_rules.yaml:1]: won't have any adverse impact

**Pull request merged**